docs/RISKS.md

```diff
@@ -0,0 +1,40 @@
+
+## Risk table
+
+| No. | Name of Risk | Description | Probabilty of Occurence (0-1) | Impact (0-10) | Total factor (Probability * Impact) | Mitigation | Person in Charge |
+| --- | --- | --- | --- | --- | --- | --- | --- |
+| 1 | Insufficient communication | With a team of five, there is a possibility of communication breakdown. This could lead to missed deadlines and errors in the project. | 0,75 | 6 | 4,5 | To mitigate the risk, there are multiple levers to pull: (a) Awareness for this risk, (b) assigning roles and responsibilites, (c) use the Discord server to document everything and allow others to step in, (c) being open for feedback | Jens |
+| 2 | Wrong time estimations | Accurately estimating the time required to complete a project is a critical aspect of project management. If the team underestimates the time required, they may not have enough time to complete the project, leading to delays and missed deadlines. | 0,75 | 5 | 3,75 | To mitigte this risk, the team draws attention to the past estimations. The tickets to estimate are broken down into smaller, more manageable and clearer tasks. They were prepared and prioritized for refinement (estimating) by the project manager. The project manager makes sure that there are no blockers or issues in sight when giving the team the task to implement. | Jens |
+| 3 | Misunderstandment of requirements | Requirements are the foundation of any project, and if they are misunderstood, it can result in a project that does not meet the intended goals and objectives. This can lead to wasted time, effort, and resources, as well as frustration among team members and the customer (Kay). | 0,3 | 5 | 1,5 | Before a ticket is put into a sprint, it is talked about in the team. Acceptance criteria are discussed to make sure the key points are clear to everyone. The team is obligated to fulfill the acceptance criteria only. | Jens, Markus, Jonas, Chris, Lars |
+| 4 | Unknown complexity when creating tickets | The scope of the project may expand beyond what the team can handle, leading to delays and possible failure. | 0,2 | 7 | 1,4 | To mitigate this risk, tickets are prioritized and broken down into smaller parts. Regular reviews deepens the understanding on complexity and technical ways to implement the features, helping the project manager to get a feeling for the involved complexity for a new ticket. | Jens |
+| 5 | Technology failures | This may include breaking changes in our tech stack requiring immediate changes, as well as unexpected behavior of technology, ultimately leading to the result that the goal of the project is not achievable. | 0,1 | 10 | 1 | This risk can be best mitigated by two strategies: (a) choosing reliable technology with a record of stability as well as reliability (following best practices), (b) training team member to employ them with the power to response to technology failure immediately on their own behalf | Jonas |
```

Comment:
b) is not clear to me. How am I supposed to train the team members? Is the goal of this task that everyone understands what each dependency in our project is used for (e.g. understand the use of each plugin in the requirements.txt or each implementation in the gradle file) ?

Reply:
This could be one of the action points you could take to manage this risk. :)

Reply:
Are there more you have in mind? I couldn't think of anything else and I dont think that this is enough to approach this risk

Reply:
e.g. gradual changes, testing thoroughly as well as a rollback plan in case of failures